src/components/assetlib_projects/asset_library_projects.tscn

```diff
@@ -0,0 +1,223 @@
+[gd_scene load_steps=2 format=3 uid="uid://c1sh1x6y1kwc2"]
+
+[ext_resource type="Script" path="res://src/components/assetlib_projects/asset_library_projects.gd" id="1_m0pc7"]
+
+[node name="AssetLibraryProjects" type="VBoxContainer"]
+anchors_preset = 15
+anchor_right = 1.0
+anchor_bottom = 1.0
+grow_horizontal = 2
+grow_vertical = 2
+script = ExtResource("1_m0pc7")
+
+[node name="SearchField" type="LineEdit" parent="."]
+unique_name_in_owner = true
+layout_mode = 2
+placeholder_text = "Search Templates, Projects, and Demos"
+
+[node name="FilterSortOptions" type="HBoxContainer" parent="."]
+layout_mode = 2
+
+[node name="Label4" type="Label" parent="FilterSortOptions"]
+layout_mode = 2
+text = "Version:"
+
+[node name="VersionOption" type="OptionButton" parent="FilterSortOptions"]
+unique_name_in_owner = true
+layout_mode = 2
+size_flags_horizontal = 3
+
+[node name="VSeparator4" type="VSeparator" parent="FilterSortOptions"]
+layout_mode = 2
+
+[node name="Label" type="Label" parent="FilterSortOptions"]
+layout_mode = 2
+text = "Sort:"
+
+[node name="SortOption" type="OptionButton" parent="FilterSortOptions"]
+unique_name_in_owner = true
+layout_mode = 2
+size_flags_horizontal = 3
+item_count = 6
+selected = 0
+popup/item_0/text = "Recently Updated"
+popup/item_0/id = 0
+popup/item_1/text = "Least Recently Updated"
+popup/item_1/id = 1
+popup/item_2/text = "Name (A-Z)"
+popup/item_2/id = 2
+popup/item_3/text = "Name (Z-A)"
+popup/item_3/id = 3
+popup/item_4/text = "License (A-Z)"
+popup/item_4/id = 4
+popup/item_5/text = "License (Z-A)"
+popup/item_5/id = 5
+
+[node name="VSeparator" type="VSeparator" parent="FilterSortOptions"]
+layout_mode = 2
+
+[node name="Label2" type="Label" parent="FilterSortOptions"]
+layout_mode = 2
+text = "Category:"
```

Comment:
please add
horizontal_alignment = 1
vertical_alignment = 1
just make the labels be centered (for this and any other in the HboxContainer)